src/libraries/System.Collections.Immutable/tests/ImmutableArrayBuilderTest.cs

```diff
@@ -257,6 +258,16 @@ public void IndexOf()
                 (b, v, i, c, eq) => b.IndexOf(v, i, c, eq));
         }
 
+        [Fact]
+        public void IndexOf_WithoutCountParam()
+        {
+            var builder = ImmutableArray.Create(1.5, 2.5, 3.5).ToBuilder();
+            var integerPartComparer = new DelegateEqualityComparer<double>(equals: (x, y) => Math.Floor(x) == Math.Floor(y));
```

Comment:
Nit: using floating point equality when it could be avoided makes me kind of nervous. Other than that test seems correct though.